src/mixins/date-mixin.ts

```diff
@@ -187,4 +208,15 @@ export default class DateMixin extends Mixins(CommonMixin) {
     const dateLocalMs = new Date(date).setHours(0, 0, 0, 0)
     return Math.round((dateLocalMs - todayLocalMs) / this.MS_IN_A_DAY)
   }
+
+  /**
+   * Padding function to ensure month, date, hour, minutes and seconds front padded with zero if input val < 10
+   * @returns '01' for 1
+   * @returns 10 for 10
+   */
+  pad (number : Number) : String {
```

Comment:
The formal parameter type should be `number` (which is a type) not `Number` (which is a class).
The return type should be fixed similarly.

Finally, you don't need this as there is already a JS equivalent: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart